src/server/exchange-controller.js

```diff
@@ -0,0 +1,20 @@
+import permissions from '../domain/permissions.js';
+import DocumentService from '../services/document-service.js';
+import needsPermission from '../domain/needs-permission-middleware.js';
+
+class ExchangeController {
+  static get inject() { return [DocumentService]; }
+
+  constructor(documentService) {
+    this.documentService = documentService;
+  }
+
+  registerApi(router) {
+    router.get('/api/v1/exchange/exports', [needsPermission(permissions.LIST_EXPORTABLE_CONTENT)], async (req, res) => {
```

Comment:
I guess here we need the API key check, right?
Which means the tickets are wrongly blocked.

Reply:
Yes, but I made it admin-accessible until we have that

Reply:
We could merge the API key PR first and do this in one go maybe. Guess that would be easier, or?

Reply:
I don't understand, why it would be easier. The API key PR just has to make sure the request is associated with a user that has this permission (either by introducing a new role, or by extending the role <-> permission mapping so that the calling user has `LIST_EXPORTABLE_CONTENT` permission.

Reply:
Ok so it's a chicken and egg problem then, whichever PR is second needs to do this correlation 😆 
Then I'll do it in mine.